packages/playground/src/components/deposit_dialog.vue

```diff
@@ -94,6 +94,8 @@ const props = defineProps({
   openDepositDialog: Boolean,
   twinId: Number,
 });
+const qrHeader = `<b> OR </b>
+      <p class="mb-3">Use ThreeFold Connect to scan this QRcode:</p>`;
```

Comment:
i think the best practice is to use [slot](https://vuejs.org/guide/components/slots),